libs/domains/clusters/feature/src/lib/cluster-delete-modal/cluster-delete-modal.tsx

```diff
@@ -0,0 +1,91 @@
+import { ClusterDeleteMode } from 'qovery-typescript-axios'
+import { useState } from 'react'
+import { BannerBox, BannerBoxEnum, IconAwesomeEnum, InputSelect, ModalConfirmation } from '@qovery/shared/ui'
+import useDeleteCluster from '../hooks/use-delete-cluster/use-delete-cluster'
+
+export interface ClusterDeleteModalProps {
+  organizationId: string
+  clusterId: string
+  name: string
+}
+
+export function ClusterDeleteModal({ organizationId, clusterId, name }: ClusterDeleteModalProps) {
+  const { mutateAsync } = useDeleteCluster()
+  const [clusterDeleteMode, setClusterDeleteMode] = useState<ClusterDeleteMode>(ClusterDeleteMode.DEFAULT)
+
+  const formattedClusterDeleteMode = (clusterDeleteMode: string) =>
+    clusterDeleteMode
+      .split('_')
+      .map((word) => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase())
```

Comment:
use `upperCaseFirstLetter` ?

Reply:
No because we want all first letters capitalize like `Qovery Cluster Default` to example